specifyweb/frontend/js_src/lib/localization/attachments.ts

```diff
@@ -97,6 +97,114 @@ export const attachmentsText = createDictionary({
     'uk-ua': 'Показати форму',
     'de-ch': 'Formular anzeigen',
   },
+  multipleFilesSelected: {
+    'en-us': 'Multiple files selected',
+  },
+  fileSize: {
+    'en-us': 'File Size',
+  },
+  deleted: {
+    'en-us': 'Deleted',
+  },
+  noFile: {
+    'en-us': 'No File',
+  },
+  incorrectFormatter: {
+    'en-us': 'Incorrectly Formatted',
+  },
+  alreadyUploaded: {
+    'en-us': 'Already Uploaded',
+  },
+  alreadyDeleted: {
+    'en-us': 'Already Deleted',
+  },
+  skipped: {
+    'en-us': 'Skipped',
+  },
+  cancelled: {
+    'en-us': 'Cancelled',
+  },
+  frontEndInterruption: {
+    'en-us': '{action:string} was in progress when interruption occured',
+  },
+
+  matchError: {
+    'en-us': 'Match Error',
+  },
+  errorReadingFile: {
+    'en-us': 'Error reading file',
+  },
+
+  unhandledFatalResourceError: {
+    'en-us': 'Unhandled fatal resource error:',
+  },
+  attachmentImportDatasetsCount: {
+    'en-us': 'Attachment Import Datasets ({count:number})',
+  },
+  timeStampCreated: {
+    'en-us': 'Timestamp Created',
+  },
+  timeStampModified: {
+    'en-us': 'Timestamp Modified',
+  },
```

Comment:
Don't create localization strings for database colunn names - use schema localization strings instead.

i.e instead of `attachmentText.timeStampCreated()` do `strictGetModel('WorkBench').strictGetField('timestampCreated').label`

see https://github.com/specify/specify7/issues/4011#issuecomment-1720382844 for longer explanation on reasons behind this (and exceptions to this rule)

@CarolineDenis @melton-jason

Reply:
fixed